test-infra/npm-shrinkwrap.json

```diff
@@ -2,19 +2,19 @@
   "name": "bootstrap",
   "version": "4.0.0-alpha",
   "npm-shrinkwrap-version": "200.4.0",
-  "node-version": "v0.12.7",
+  "node-version": "v4.1.0",
```

Comment:
This could lead to problems with native libs since this version of node isn't available on Travis yet.